src/Platform.Handlers/src/Xamarin.Platform.Handlers/Handlers/Label/LabelHandler.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+#if __IOS__
+using NativeView = UIKit.UILabel;
```

Comment:
Is this cool? Annoying? 

This way we basically have on handler that just creates the default functions and then the extension methods are where the behavior splits

Reply:
annoying, but a necessary evil.

Reply:
Yea that was the general consensus among the team :-)